src/qt/rpcconsole.cpp

```diff
@@ -805,7 +805,7 @@ void RPCConsole::clear(bool clearHistory)
     QString clsKey = "Ctrl-L";
 #endif
 
-    message(CMD_REPLY, (tr("Welcome to the %1 RPC console.").arg(PACKAGE_NAME) + "<br>" +
+    message(CMD_REPLY, (tr("Welcome to the %1 Node console.").arg(PACKAGE_NAME) + "<br>" +
```

Comment:
in commit 6cdb9560dc8d301fb96cdb90c2b7c3d6fd0e4399:

I think the console is still all about RPC, so could keep the previous name?

Reply:
I assumed changing the message from 'RPC console' to 'Node console' would make the new window name ('Node window') consistent with the welcome message.

Reply:
The "Node window" has 4 tabs, the RPC console is one of those.

Reply:
Let me change that